Kudu.Services/Functions/FunctionController.cs

```diff
@@ -22,6 +25,100 @@ public FunctionController(IFunctionManager manager, ITraceFactory traceFactory)
             _traceFactory = traceFactory;
         }
 
+        [HttpPut]
```

Comment:
Just curious: Why do we have these http verb attributes in addition to the HttpMethodConstraints above? Isn't one or the other sufficient?